packages/brick-container/src/styles/theme/default.css

```
@@ -0,0 +1,115 @@
+:root,
+[data-override-theme="light"] {
+  /* Basic CSS variables Refs: https://tapd.easyops.local/pages/viewpage.action?pageId=36045937 */
+  --color-brand: #0071eb;
+  --color-brand-hover: var(--light-palette-blue-4);
+  --color-brand-active: var(--light-palette-blue-8);
+  --color-fill-global-bg-1: #f7f8fa;
+  --color-fill-global-bg-2: var(--light-palette-gray-1);
+  --color-fill-bg-container-L1: var(--light-palette-gray-1);
```

不要混用大小写，这里 `l1` 不好区分就直接写成 `1` 吧。
```suggestion
  --color-fill-bg-container-1: var(--light-palette-gray-1);
```